JS Basics/intro.js

```diff
@@ -0,0 +1,20 @@
+let sentence = "I can eat bananas all day";
+let word = sentence.slice(10, 17);
+alert(word.toUpperCase());
+
+let cars = ["Saab", "Volvo", "BMW"];
+
+let bmw = cars[2];
+console.log(bmw);
+
+cars[0] = "Mercedes";
+console.log(cars);
+
+cars.pop();
+console.log(cars);
+
+cars.push("Audi");
```

Comment:
Just a FYI, no need to change: I would also suggest another option to create a new array without modifying existing: [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_array_literals). It's really comfortable when you need to work with immutable entities, meanwhile `push` mutates original array and returns numeric value with new array length